README.md

```diff
@@ -354,6 +355,10 @@ Table of Contents
    * https://codio.com/ - Codio is a cloud-based computer programming platform for universities, schools, and developer professionals.
    * http://www.stackhive.com/ - Cloud based IDE in browser that supports HTML5/CSS3/jQuery/Bootstrap
 
+## API Management
+
+  * http://apigee.com - Apigee Developer unlimited trial 
```

Comment:
its already listed in the BaaS section which seems a good place for it so gonna close this one.  when I make the website for this data I'll allow tags and multi catagories etc but for now with the giant-readme format, I'd prefer not to list them twice